firmware/application/irq_controls.cpp

```diff
@@ -162,16 +162,12 @@ static bool switches_update(const uint8_t raw) {
 }
 
 static bool encoder_update(const uint8_t raw) {
-    bool encoder_changed = false;
-
-    encoder_changed |= encoder_debounce[0].feed((raw >> 6) & 0x01);
-    encoder_changed |= encoder_debounce[1].feed((raw >> 7) & 0x01);
-
-    return encoder_changed;
+    // TODO: swap +1/-1 directions in encoder.update() to avoid needless swizzing of phase bits here
+    return encoder_debounce.feed(((raw >> 7) & 0x01) | ((raw >> 5) & 0x02));
```

Comment:
Note to reviewers:  This "should" simply be "raw >> 6" but the bits would be swapped versus the original code, thereby resulting in the encoder dial direction being reversed.  I'd like to fix that later to keep this PR simpler.